src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -152,6 +158,22 @@
       <Name>System.Globalization</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.7'">
```

Comment:
I agree we should probably keep these here until we decide we don't care about testing the older contract versions.